misc/cicd/README.md

```diff
@@ -0,0 +1,163 @@
+# Sample Configurations to improve ABAP deployment task
+
+Please refer to the following guide for more info;
+
+https://www.npmjs.com/package/@sap/ux-ui5-tooling
+
+## Using environment variables using .env file
+
+Create a .env file in the root of your SAPUI5 project and append the following content;
+```
+XYZ_USER=myusername
+XYZ_PASSWORD=mypassword
+```
+
+Update `ui5-deploy.yaml` with the `credentials` node;
+
+```YAML
+    configuration:
+      yes: true
+      failFast: true
+      target:
+        url: https://XYZ.sap-system.corp:44311
+        client: 200        
+      credentials:
+        username: env:XYZ_USER
+        password: env:XYZ_PASSWORD
+```
+
+When you run `npm run deploy` or `npm run undeploy`, it will pickup the environment variables.
+
+Setting `env:` for the credentials is only required if you want to configure a `.env` file to load environment variales, otherwise you just export them as normal;
+
+```bash
+export XYZ_USER='~username'
+export XYZ_PASSWORD='~password''
+```
+
+`ui5-deploy.yaml` is then updated as 
+```YAML
+      credentials:
+        username: XYZ_USER
+        password: XYZ_PASSWORD
+```
+Or if using CLI params;
+```bash
+--username XYZ_USER --pasword XYZ_PASSWORD
+```
+
+Additional note, if you are using a CI/CD pipeline, then you can make further updates to your `ui5-deploy.yaml` as shown above;
+- Add `yes: true` to by-pass the `Yes` confirmation prompt
+- Add `failFast: true` to immediately exist the process if any exception is thrown, for example, a typical scenario is where authentication might fail and you want to disable the credentials prompts from being shown. This will exit with code of `1`.
```

Comment:
That should be "immediately exit the process"

Reply:
Updated now.